packages/destination-actions/src/destinations/stackadapt-audiences/index.ts

```diff
@@ -62,6 +62,24 @@ const destination: DestinationDefinition<Settings> = {
       }
     }
   },
+  onDelete: async (request, { payload }) => {
```

Comment:
Curious if there ss the a performBatch we can use for delete just like profile sync? Not sure batch delete is a supported action on Segment Engage.

Reply:
I haven't seen any indication batch delete is supported in the docs or in the other destinations that implement `onDelete`.